pkg/controller/kubedirectorcluster/types.go

```diff
@@ -89,10 +89,9 @@ const (
 	rm -rf /opt/guestconfig/appconfig.tgz`
 	appPrepConfigStatus = "/opt/guestconfig/configure.status"
 	appPrepConfigRunCmd = `rm -f /opt/guestconfig/configure.* &&
-	echo -n %s= > ` + appPrepConfigStatus + ` &&
-	nohup sh -c "` + appPrepStartscript + ` --configure;
-	echo -n $? >> ` + appPrepConfigStatus + `" > /opt/guestconfig/configure.stdout  
-	2> /opt/guestconfig/configure.stderr  &`
+	echo -n %s= > ` + appPrepConfigStatus + ` && 
+	nohup sh -c "` + appPrepStartscript + ` --configure 2>/opt/guestconfig/configure.stderr 1>/opt/guestconfig/configure.stdout;
+	echo $? >> ` + appPrepConfigStatus + `" &`
```

Comment:
Was removing the "-n" option here intentional?

Reply:
Yes, I think it's not required since line 92 writes 
```
docker://789aabbca73481a9b0b2f0891281cd32e05995353287a9e1b07810bcf6d1417c=0
```
which is what we need

Reply:
Right, but previously we didn't have a trailing carriage return after all that. It may not matter, depends on the processing code, but just wondering why you put that final carriage return back in.

Reply:
On line 93 ?
For readability, it's basically split into 3 subcommands 

line 92 - write container id in status
line 93 - run startscript in the background and write stdout and stderr in respective files
line 94 - update status file with exit code of the previous command.

Reply:
Yeah, it's the change in line 94 that I'm asking about.

Reply:
ok, I see this is the original question of why did we remove "-n". Actually it is not making any diff in status. I always see docker://789aabbca73481a9b0b2f0891281cd32e05995353287a9e1b07810bcf6d1417c=0
We can add -n back.

But there is a new problem that I found. With nohup the $? is always returning 0.
Trying to debug that

Reply:
Added -n back, and replaced double quote with a single.

This now works as expected

Reply:
Are we sure. :-)

Man this particular code has been ornery.

Reply:
Are we sure. :-)

Man this particular code has been ornery.